src/Operation/ListIndexes.php

```diff
@@ -144,6 +144,9 @@ private function executeCommand(Server $server): IndexInfoIteratorIterator
 
         $cursor->setTypeMap(['root' => 'array', 'document' => 'array']);
 
-        return new IndexInfoIteratorIterator(new CachingIterator($cursor), $this->databaseName . '.' . $this->collectionName);
+        /** @var CachingIterator<int, array> $iterator */
```

Comment:
Is it necessary to repeat the variable name in these annotations? I have some recollection of _not_ doing so previously.

Reply:
It's unnecessary when used on a property, but required in the code.